snuba/web/rpc/timeseries.py

```diff
@@ -0,0 +1,181 @@
+import uuid
+from datetime import datetime
+
+from google.protobuf.json_format import MessageToDict
+from sentry_protos.snuba.v1alpha.endpoint_aggregate_bucket_pb2 import (
+    AggregateBucketRequest,
+    AggregateBucketResponse,
+)
+
+from snuba.attribution.appid import AppID
+from snuba.attribution.attribution_info import AttributionInfo
+from snuba.datasets.entities.entity_key import EntityKey
+from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.pluggable_dataset import PluggableDataset
+from snuba.query import SelectedExpression
+from snuba.query.conditions import combine_and_conditions, combine_or_conditions
+from snuba.query.data_source.simple import Entity
+from snuba.query.dsl import Functions as f
+from snuba.query.dsl import (
+    NestedColumn,
+    and_cond,
+    column,
+    in_cond,
+    literal,
+    literals_array,
+)
+from snuba.query.expressions import Expression, FunctionCall
+from snuba.query.logical import Query
+from snuba.query.query_settings import HTTPQuerySettings
+from snuba.request import Request as SnubaRequest
+from snuba.utils.metrics.timer import Timer
+from snuba.web.query import run_query
+
+_HARDCODED_MEASUREMENT_NAME = "eap.measurement"
+
+
+def _get_measurement_field(
+    request: AggregateBucketRequest,
+) -> Expression:
+    field = NestedColumn("attr_num")
+    # HACK
+    return field[request.metric_name]
+
+
+def _treeify_or_and_conditions(query: Query) -> None:
```

Comment:
Can we use the existing functions for this? `snuba.query.conditions.combine_or_conditions`?